src/sage/calculus/transforms/dft.py

```diff
@@ -71,20 +71,20 @@
 #
 #                  https://www.gnu.org/licenses/
 ##########################################################################
-from sage.rings.number_field.number_field import CyclotomicField
+from sage.functions.all import sin, cos
 from sage.misc.lazy_import import lazy_import
```

Comment:
Is this (importing from `all`) all right?

Reply:
It's not enforced for sage.functions yet, but I've changed it